src/notation/internal/notationplayback.cpp

```diff
@@ -258,7 +258,7 @@ void NotationPlayback::makeChunk(midi::Chunk& chunk, tick_t fromTick) const
 
         midi::EventType etype = static_cast<midi::EventType>(ev.type());
         static const std::set<EventType> SKIP_EVENTS
-            = { EventType::ME_INVALID, EventType::ME_EOT, EventType::ME_TICK1, EventType::ME_TICK2 };
+            = { EventType::ME_INVALID, EventType::ME_EOT };
```

Comment:
Even though ME_TICK1/ME_TICK2 aren't standard MIDI events, we still need to know about them - currently there's no playback for them supported at all, but at least it will trigger tickPlayed events which triggers the playback cursor movement etc.